client/webserver/site/src/css/main.scss

```diff
@@ -3,7 +3,7 @@ html {
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased !important;
   -moz-font-smoothing: antialiased !important;
-  text-rendering: optimizelegibility !important;
+  text-rendering: optimizeLegibility !important;
```

Comment:
Nice catch.

Reply:
Updated eslint caught it I think, but thanks. :grin: